modules/v2x/v2x_proxy/os_interface/BUILD

```diff
@@ -17,6 +17,7 @@ cc_library(
         "//modules/v2x/proto:v2x_obu_rsi_cc_proto",
         "//modules/v2x/proto:v2x_obu_traffic_light_cc_proto",
         "//modules/v2x/proto:v2x_traffic_light_cc_proto",
+        "//modules/v2x/proto:v2x_obstacles_cc_proto",
```

Comment:
Notice the order, you can fix this by `./apollo.sh format -b modules/v2x/v2x_proxy/os_interface`